compliance/fnms/fnms_low_licenses_available/fnms-low-available-licenses.pt

```diff
@@ -0,0 +1,166 @@
+name "FlexNet Manager Low Available Licenses"
+rs_pt_ver 20180301
+type "policy"
+short_description "Looks up Flexnet Manager Licenses and finds all Flexnet Manager Licenses with available count less than user provide percentage, and sends the result as an email.  See the [README](https://github.com/flexera/policy_templates/tree/master/compliance/fnms/fnms_low_licenses_available) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
+severity "medium"
+category "Compliance"
+info(
+  version: "2.1",
+  provider: "Flexera FNMS",
+  service: "",
+  policy_set: "Low License Count"
+)
+
+parameter "param_fnms_url" do
+  type "string"
+  label "FNMS Report URL"
+  description "Full FlexNet URL (e.g. https://demo.flexnetmanager.com/Suite )"
+  allowed_pattern /^(http:\/\/|https:\/\/)+[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z0-9]{2,5}(:[0-9]{1,5})?(\/.*)?$/
+end
+
+parameter "param_available_watermark" do
+  type "number"
+  label "Percentage Available"
+  description "Percentage threshold of available licenses"
+end
+
+parameter "param_report_id" do
+  type "string"
+  label "FNMS Report ID"
+  description "FlexNet Manager Custom View ID"
+end
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+  description "A list of email addresse(s) to notify"
+end
+
+credentials "api_auth" do
+  schemes "api_key","ntlm"
+  label "FNMS"
+  description "Select the FNMS Token Credential from the list."
```

Comment:
I think Token should be removed since it's just the Credential, it can be the token or username/password.